keyboards/ergodox_extended/info.json

```diff
@@ -0,0 +1,144 @@
+{
+    "manufacturer": "Gephorian",
+    "keyboard_name": "ergodox_extended",
+    "maintainer": "Gephorian",
+    "bootloader": "atmel-dfu",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "processor": "atmega32u4",
+    "url": "https://github.com/Gephorian/ergodox-extended",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+    },
+    "community_layouts": ["ergodox"],
+    "layouts": {
+        "LAYOUT_ergodox": {
```

Comment:
Do not use stacked layouts.

Reply:
Removed.

Reply:
Not what I meant, but it would have been necessary anyway. You need to move the right half of the layout to the right side.

Reply:
Oh, you mean in `keymaps/default/keymap.c`?

Reply:
No, the layout in general.

Reply:
Have a look at my most recent commit. Is that what you meant I needed to do? If not, can you link me to an example?

Reply:
Looks better. Please also run `qmk format-json -i` on this file.

Reply:
Layout needs to be adjusted.
![image](https://github.com/qmk/qmk_firmware/assets/4781841/248a6082-7380-4aed-abfc-ac787c74f45c)

Reply:
Done. Let me know if there's anything else that needs to be done with layout.

Reply:
Is there any way to render the keymap layouts locally?

Reply:
Press Ctrl+Shift+I in the Configurator. You can also verify the ordering and matrix positions are correct there.

Reply:
Thank you, that helped a lot! Got the layout looking a bit more spiffy.